pyproject.toml

```diff
@@ -24,6 +24,7 @@ dependencies = [
     "freetype-py",
     "uharfbuzz",
     "jinja2",
+    "hsluv==5.0.0",
```

Comment:
Can you explain why you pinned it to 5.0.0 exactly? I see 5.0.4 is available for example...

Reply:
there was no particular reason

Reply:
Let's pin to "hsluv==5".

Reply:
Let's use ~=5 to be a little more flexible

Reply:
updated with a range to be consistent with previous libs in toml